README.md

```diff
@@ -22,7 +22,8 @@ __Windows__
 * [Visual Studio 2015](https://www.visualstudio.com/en-us/downloads/download-visual-studio-vs.aspx)
 * [Visual C++ Redistributable Packages for Visual Studio 2015](http://www.microsoft.com/en-us/download/details.aspx?id=48145)
 * [Cmake 3.1.0+](http://www.cmake.org/download/) (add to PATH)
-* [Python 2.7+](https://www.python.org/downloads/) (add to PATH)
+* [Python](https://www.python.org/downloads/) (add to PATH)
```

Comment:
@tambry I speak of version 2.7 and 3.5.
In the "Pull Request" I explained above.
and logically, people click on the link Python, and thus comes to the site, and it can take one he wants.
that's why I is deleted.